documentation/src/main/asciidoc/topics/proc_cli_batching_files.adoc

```diff
@@ -36,3 +36,5 @@ hello
 [//containers/default]> disconnect
 [disconnected]>
 ----
+
+NOTE: CLI batch files support system property expansion: strings of the form `${property}` will be replaced with the value of the `property` system property.
```

Comment:
just a nit, but from the [doc guidelines](https://infinispan.org/docs/dev/titles/contributing/contributing.html#documentation_guidelines_format_reference):

Encapsulate notes, tips, and warnings in brackets. Surround the text in four equals characters, as follows:

    [NOTE]
    ====
    This is a note that you should read.
    ====